src/rust-crypto/rust-crypto.ts

```diff
@@ -229,6 +235,23 @@ export class RustCrypto implements CryptoBackend {
         }
     }
 
+    /** called by the sync loop to process
+     *
+     * @param syncData - Object containing sync tokens associated with this sync
+     * @param deviceLists - device_lists field from /sync, or response from
+     */
+    public async processDeviceLists(
+        syncData: ISyncStateData,
+        deviceLists: Required<ISyncResponse>["device_lists"],
+    ): Promise<void> {
+        // Initial syncs don't have device change lists. We'll either get the complete list
+        // of changes for the interval or will have invalidated everything in willProcessSync
+        if (!syncData.oldSyncToken || !deviceLists) return;
```

Comment:
I add the same verification that in the old crypto but I don't know if it's relevant here.